webapp/src/components/home/homeContainer.js

```diff
@@ -50,7 +51,7 @@ export default class HomeContainer extends React.Component {
       }
       });
 
-    fetch(webAppConfig.api.url+'/activity',myInit)
+    fetch(webAppConfig.api.url + '/activity?count=20',myInit)

```

Comment:
@jamesbak - do you want a scrollable view for the check ins panel, or should it not have a scroll at all and just show the amount of check ins that fit in the panel?